### ISSUE TITLE
Why enable the browser to have two tabs and not open the web properly?

### ISSUE THREAD

**Issue by smp12345** [go-rod/rod#306]
**Rod Version:** v0.84.3
chrome Version: 版本 87.0.4280.88（正式版本）
go Version ： go1.14.4 windows/amd64
Window7

sample code：

```go
package main

import (
	"github.com/go-rod/rod"
	"time"
)

func main() {
	page := rod.New().MustConnect().MustPage("https://www.baidu.com/")
	page.MustWaitLoad().MustScreenshot("a.png")
	time.Sleep(time.Hour)
}
```

When running the code above, chrome will open two browser window directly, one of the Windows web site: file:///C:/Users/ADMINI~1/AppData/Local/Temp/rod/self-close.html, the other is a blank page, baidu.com is not normal to open

Help me, thank you
![1](https://user-images.githubusercontent.com/50661521/101848909-915cb200-3b91-11eb-840b-d4c4b6d8cd2f.png)
![2](https://user-images.githubusercontent.com/50661521/101848914-9588cf80-3b91-11eb-924c-47f0ad04aa6d.png)

**Comment by smp12345** [comment]
I use the sample code of the official website. After opening the browser, two tabs will be opened directly. My computer is Windows 7, and the test on Ubuntu last night is normal, please refer to the screenshot above.What other information do I need to provide?

**Comment by ysmood** [comment]
Works fine with Windows10, sorry I don't know how to reproduce it:

![Kapture 2020-12-11 at 10 38 48](https://user-images.githubusercontent.com/1415488/101850502-33ce6280-3b9d-11eb-9815-4062cee1bce4.gif)

**Comment by ysmood** [comment]
Can you update your chrome to the latest version and try again?

**Comment by smp12345** [comment]
What's the meaning this path:  file:///C:/Users/ADMINI~1/AppData/Local/Temp/rod/self-close.html   ?

**Comment by smp12345** [comment]
I'm using Chrome 87, which should be up to date

**Comment by ysmood** [comment]
> What's the meaning this path: file:///C:/Users/ADMINI~1/AppData/Local/Temp/rod/self-close.html ?

It's the default `StartURL`, please read the doc:

https://github.com/go-rod/rod/blob/5ebed67638c56892dd3822268456fdf1aa9f05da/lib/launcher/launcher.go#L41-L46

**Comment by smp12345** [comment]
I tested ubuntu last night and saw only a TAB, not this one, just go to Baidu.com

**Comment by ysmood** [comment]
Can you try this code?

```go
package main

import (
	"time"

	"github.com/go-rod/rod"
	"github.com/go-rod/rod/lib/launcher"
)

func main() {
	u := launcher.New().StartURL("about:blank").MustLaunch()
	page := rod.New().ControlURL(u).MustConnect().MustPage("https://www.baidu.com/")
	page.MustWaitLoad().MustScreenshot("a.png")
	time.Sleep(time.Hour)
}
```

**Comment by smp12345** [comment]
![1](https://user-images.githubusercontent.com/50661521/101851573-cae3ec00-3b96-11eb-969e-d899dae50bc6.png)
![2](https://user-images.githubusercontent.com/50661521/101851587-cf100980-3b96-11eb-8eb1-cbc2e8b43090.png)
Open two blank pages, there is no url I sent above

**Comment by ysmood** [comment]
Seems like Chrome crashed, not sure about why is happening. I don't have Window7 to test it, not sure how to help.

**Comment by smp12345** [comment]
Can you install a virtual machine and test it out?Thank you very much.After all, Windows 7 still has a large user base

**Comment by ysmood** [comment]
Sure, I just don't time for it now, I will check when I have time. Thank you for your report!

Can you share more debug info? To enable it is easy just add one line to the `.rod` file, so you `.rod` looks like:

```txt
show
cdp
```

Then provide your console log to us.

**Comment by ysmood** [comment]
Can you run this:

```go
package main

import (
	"time"

	"github.com/go-rod/rod"
	"github.com/go-rod/rod/lib/launcher"
)

func main() {
	u := launcher.NewUserMode().MustLaunch()
	page := rod.New().ControlURL(u).MustConnect().MustPage("https://www.baidu.com/")
	page.MustWaitLoad().MustScreenshot("a.png")
	time.Sleep(time.Hour)
}
```

**Comment by smp12345** [comment]
![3](https://user-images.githubusercontent.com/50661521/101852707-39c24480-3b99-11eb-8575-7c679371cb0a.png)
There is only one blank page after loading

**Comment by smp12345** [comment]
首先有2个空白页，然后会自动关闭一个空白页

**Comment by smp12345** [comment]
There are two blank pages first, and then a blank page is automatically closed

**Comment by smp12345** [comment]
I tested it on another win7 and had exactly the same problem.

**Comment by smp12345** [comment]
[cdp] 2020/12/11 10:15:38 => #1 @00000000 Target.setDiscoverTargets {"discover":true}
[cdp] 2020/12/11 10:15:38 <- @00000000 Target.targetCreated {"targetInfo":{"targetId":"6BDCBC362A05C18BEAA41C74A54465BA","type":"page","title":"about:blank","url":"about:blank","attach
ed":false,"canAccessOpener":false,"browserContextId":"F0B2995C213DC8CD63BEB49FBB634899"}}
[cdp] 2020/12/11 10:15:38 <- @00000000 Target.targetCreated {"targetInfo":{"targetId":"77324c01-a277-4a70-a930-75a430687c9a","type":"browser","title":"","url":"","attached":true,"canAc
cessOpener":false}}
[cdp] 2020/12/11 10:15:38 <= #1 {}
[cdp] 2020/12/11 10:15:38 => #2 @00000000 Browser.getVersion {}
[cdp] 2020/12/11 10:15:38 <= #2 {"protocolVersion":"1.3","product":"Chrome/88.0.4298.0","revision":"@e62cb7e3fc7c40548cef66cdf19d270535d9350b","userAgent":"Mozilla/5.0 (Windows NT 6.1;
 WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4298.0 Safari/537.36","jsVersion":"8.8.134"}
[cdp] 2020/12/11 10:15:38 => #3 @00000000 Target.createTarget {"url":"about:blank"}
[cdp] 2020/12/11 10:15:38 <- @00000000 Target.targetCreated {"targetInfo":{"targetId":"6DEE2AE6437F6117745FA0BAE58E520E","type":"page","title":"","url":"about:blank","attached":false,"
canAccessOpener":false,"browserContextId":"F0B2995C213DC8CD63BEB49FBB634899"}}
[cdp] 2020/12/11 10:15:38 <= #3 {"targetId":"6DEE2AE6437F6117745FA0BAE58E520E"}
[cdp] 2020/12/11 10:15:38 <- @00000000 Target.targetInfoChanged {"targetInfo":{"targetId":"6DEE2AE6437F6117745FA0BAE58E520E","type":"page","title":"about:blank","url":"about:blank","at
tached":false,"canAccessOpener":false,"browserContextId":"F0B2995C213DC8CD63BEB49FBB634899"}}
[cdp] 2020/12/11 10:15:38 => #4 @00000000 Target.attachToTarget {"targetId":"6DEE2AE6437F6117745FA0BAE58E520E","flatten":true}
[cdp] 2020/12/11 10:15:38 <- @00000000 Target.targetInfoChanged {"targetInfo":{"targetId":"6DEE2AE6437F6117745FA0BAE58E520E","type":"page","title":"about:blank","url":"about:blank","at
tached":true,"canAccessOpener":false,"browserContextId":"F0B2995C213DC8CD63BEB49FBB634899"}}
[cdp] 2020/12/11 10:15:38 <- @00000000 Target.attachedToTarget {"sessionId":"08B25FD9ABEE6B094F3B60E02060BA9A","targetInfo":{"targetId":"6DEE2AE6437F6117745FA0BAE58E520E","type":"page"
,"title":"about:blank","url":"about:blank","attached":true,"canAccessOpener":false,"browserContextId":"F0B2995C213DC8CD63BEB49FBB634899"},"waitingForDebugger":false}
[cdp] 2020/12/11 10:15:38 <= #4 {"sessionId":"08B25FD9ABEE6B094F3B60E02060BA9A"}
[cdp] 2020/12/11 10:15:38 => #5 @08B25FD9 Page.enable {}
[cdp] 2020/12/11 10:15:38 <= #5 {}
[cdp] 2020/12/11 10:15:38 => #6 @08B25FD9 DOM.enable {}
[cdp] 2020/12/11 10:15:38 <= #6 {}
[cdp] 2020/12/11 10:15:38 => #7 @08B25FD9 Runtime.evaluate {"expression":"window"}
[cdp] 2020/12/11 10:15:38 <= #7 {"result":{"type":"object","className":"Window","description":"Window","objectId":"-3064364799009828676.1.1"}}
panic: invalid character '.' after top-level value

**Comment by ysmood** [comment]
Can you kill all Chrome processes then try my last code again?

**Comment by smp12345** [comment]
![4](https://user-images.githubusercontent.com/50661521/101853420-865a4f80-3b9a-11eb-9b95-698fc99366ae.png)
我杀掉所有chrome进程，但还是一样的问题

**Comment by smp12345** [comment]
![4](https://user-images.githubusercontent.com/50661521/101853530-b43f9400-3b9a-11eb-88a5-a2c394803634.png)
I killed all the Chrome processes, but it's the same problem

**Comment by ysmood** [comment]
Does headless mode work?

**Comment by smp12345** [comment]
It also doesn't work properly and doesn't see the PNG file

**Comment by smp12345** [comment]
[cdp] 2020/12/11 10:28:06 => #1 @00000000 Target.setDiscoverTargets {"discover":true}
[cdp] 2020/12/11 10:28:06 <- @00000000 Target.targetCreated {"targetInfo":{"targetId":"B39B5B2457C6E8FC18E02CEB663AAF9F","type":"page","title":"","url":"file:///C:/Users/ADMINI~1/AppDa
ta/Local/Temp/rod/self-close.html","attached":false,"canAccessOpener":false,"browserContextId":"C2446520E703393410B76B87E136941F"}}
[cdp] 2020/12/11 10:28:06 <- @00000000 Target.targetCreated {"targetInfo":{"targetId":"d16a7d51-ff3f-480c-a7ba-c7d9c0a1deec","type":"browser","title":"","url":"","attached":true,"canAc
cessOpener":false}}
[cdp] 2020/12/11 10:28:06 <= #1 {}
[cdp] 2020/12/11 10:28:06 => #2 @00000000 Browser.getVersion {}
[cdp] 2020/12/11 10:28:06 <= #2 {"protocolVersion":"1.3","product":"Chrome/88.0.4298.0","revision":"@e62cb7e3fc7c40548cef66cdf19d270535d9350b","userAgent":"Mozilla/5.0 (Windows NT 6.1;
 WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4298.0 Safari/537.36","jsVersion":"8.8.134"}
[cdp] 2020/12/11 10:28:06 => #3 @00000000 Target.createTarget {"url":"about:blank"}
[cdp] 2020/12/11 10:28:06 <- @00000000 Target.targetCreated {"targetInfo":{"targetId":"48E7A8EB93808473088525BE454C0FD2","type":"page","title":"","url":"about:blank","attached":false,"
canAccessOpener":false,"browserContextId":"C2446520E703393410B76B87E136941F"}}
[cdp] 2020/12/11 10:28:07 <= #3 {"targetId":"48E7A8EB93808473088525BE454C0FD2"}
[cdp] 2020/12/11 10:28:07 => #4 @00000000 Target.attachToTarget {"targetId":"48E7A8EB93808473088525BE454C0FD2","flatten":true}
[cdp] 2020/12/11 10:28:07 <- @00000000 Target.targetInfoChanged {"targetInfo":{"targetId":"48E7A8EB93808473088525BE454C0FD2","type":"page","title":"","url":"about:blank","attached":tru
e,"canAccessOpener":false,"browserContextId":"C2446520E703393410B76B87E136941F"}}
[cdp] 2020/12/11 10:28:07 <- @00000000 Target.attachedToTarget {"sessionId":"23DC35CEC9FC7963C3E0FE7B5EE67DB8","targetInfo":{"targetId":"48E7A8EB93808473088525BE454C0FD2","type":"page"
,"title":"","url":"about:blank","attached":true,"canAccessOpener":false,"browserContextId":"C2446520E703393410B76B87E136941F"},"waitingForDebugger":false}
[cdp] 2020/12/11 10:28:07 <= #4 {"sessionId":"23DC35CEC9FC7963C3E0FE7B5EE67DB8"}
[cdp] 2020/12/11 10:28:07 => #5 @23DC35CE Page.enable {}
[cdp] 2020/12/11 10:28:07 <- @00000000 Target.targetInfoChanged {"targetInfo":{"targetId":"48E7A8EB93808473088525BE454C0FD2","type":"page","title":"about:blank","url":"about:blank","at
tached":true,"canAccessOpener":false,"browserContextId":"C2446520E703393410B76B87E136941F"}}
[cdp] 2020/12/11 10:28:07 <= #5 {}
[cdp] 2020/12/11 10:28:07 => #6 @23DC35CE DOM.enable {}
[cdp] 2020/12/11 10:28:07 <= #6 {}
[cdp] 2020/12/11 10:28:07 => #7 @23DC35CE Runtime.evaluate {"expression":"window"}
[cdp] 2020/12/11 10:28:07 <= #7 {"result":{"type":"object","className":"Window","description":"Window","objectId":"-5356785989425377739.1.1"}}
panic: invalid character '.' after top-level value

**Comment by ysmood** [comment]
Thank you, I think it's enough now, I will test Win7 later. I missed this OS because seldom developers nowadays still use Win7.

**Comment by smp12345** [comment]
Thank you. This is my first time using this framework, and I feel the documentation is excellent.I used Chromedp before, and it worked fine, but It didn't feel stable enough to fail when I clicked on an element multiple times

**Comment by smp12345** [comment]
Although the development of people using Win7 is not much, but the user use Win7 or a lot of my customers are Win7, Win7 this system or do not give up, thank you

**Comment by ysmood** [comment]
Published the v0.84.4, should be fixed.

**Comment by smp12345** [comment]
Thank you very much

**Comment by smp12345** [comment]
@ysmood After your repair, the Baidu website can be normally opened, but there is an extra blank page. How to remove this blank page
![Uploading 2.JPG…]()

**Comment by ysmood** [comment]
@smp12345 Don't remove it, the reason is the last line of here: https://pkg.go.dev/github.com/go-rod/rod@v0.85.4/lib/launcher#New

**Comment by smp12345** [comment]
I see what you mean, but in this case, the user experience is not very good. Can We use the MustPage url as an active page?

**Comment by smp12345** [comment]
I mainly use this framework for automation, it would be better to automate the interface, I need to see the process of execution, but there is an extra blank page, it doesn't look comfortable @ysmood

**Comment by ysmood** [comment]
This commit should fixed your issue 475b6597a4181a8d0c8404874f2f2d141fe9c1f7

Please check v0.85.6

**Comment by smp12345** [comment]
That's OK. Thank you very much
